### DISCUSSION TITLE
Gordian Server GUI after restart "frozen"

### DISCUSSION THREAD

**Discussion by henkvancann** [BlockchainCommons/Gordian#76] [Feature Requests]
Testing Gordian Server on Macos Version 0.1.4. I accidentally restarted the Mac without properly stopping the Test server (fully sync) and connected to iPhone (with QR).

What happened: Windows restored after restart, also Gordian Server. But sync / update not available / not working. Quiting and starting Gordian Server: no different behaviour. 

When choosing 'Install' again, that works, and the the blockchain is 'synced'.

My question: Is this expected behaviour, or is something wrong. I think we GS should be able to recover from a Mac restart without properly closing the GS app?

**Comment by Fonta1n3** [comment]
Hey,

Restarting should not cause any issues.

Can you share the bitcoin core log?

**Reply by henkvancann** [reply]
[debug.log](https://github.com/BlockchainCommons/Gordian/files/5436131/debug.log)

The second time I tried to reproduce a sudden restart, I happened to see this dialog -> 
<img width="482" alt="Screenshot 2020-10-25 at 21 04 23" src="https://user-images.githubusercontent.com/479356/97122140-d339b200-1723-11eb-9ff4-ff208ebe40ef.png">

**Reply by henkvancann** [reply]
which is good, but still if I simply power off there's still a challenge to leave Gordian Server in an acceptable state after a restart

**Comment by Fonta1n3** [comment]
Nothing to do with Bitcoin Core. 

I think its only bc Gordian Server refreshes its data every 15 seconds. There really is no point it leaving it open, Tor and Bitcoin Core will run irregardless of whether the app is open or not.

btw these types of issues are much better to be raised under issues.

**Reply by henkvancann** [reply]
My initial question: "Is this expected behaviour, or is something wrong?" If I know for sure something is an issue, I put it under issues.

Discussion or issue, still in the dark about what happened, but I'll rest my case ;)
